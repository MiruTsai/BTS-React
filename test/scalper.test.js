import React from "react";
import Scalper from "../src/components/scalper";
import renderer from "react-test-renderer";

it("render correctly",()=>{
    const tree = renderer
    .create(<Scalper />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
