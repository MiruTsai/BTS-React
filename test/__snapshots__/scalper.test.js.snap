// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render correctly 1`] = `
<div>
  <div
    className="errBoard"
  >
    <img
      className="scalperPic"
      src="../../img/hate/hate.gif"
    />
    <div
      className="alert"
    >
      你是黃牛吧！
      <br />
      這裡不歡迎你！！
    </div>
  </div>
</div>
`;
